PKI: Add support for signature_bits param to the intermediate/generate api (#17388)
hashicorp/vault: website/content/docs/secrets/pki/considerations.mdx

restrictions on signature parameters allowed by this certificate, but Go and
Vault do not support adding such restrictions.

> At this time Go lacks support for generating CSRs that are PSS signed. If

Reviewer: ```suggestion
At this time Go lacks support for CSRs with the PSS signature algorithm. If
```

It lacks more than just generating support, also lacks verification and parsing (SPKI is left Raw and not parsed).

Author: Thanks, applied the fix in the PR feedback commit.
